specification/automation/resource-manager/readme.md

```diff
@@ -59,6 +59,34 @@ input-file:
 ```
 
 
+### Tag: package-2017-05-preview

```

Comment:
Do you want to add this tag to the Basic Configuration section???

Reply:
Yes, noticed that other resource providers add tag per version (which seems to help in constructing a version by composing resources from different directories - which I wanted to do). Let me know if there is any undesirable side effect of this as I'm pretty new to swagger world.

Reply:
@finiteattractor To explain the process, You create a swagger file and you have a configuration file (README.md) where you specify the tags which lists the swagger files you would like to use for that specific tag and also specify which tag you would like to use (There can be multiple tags) for the generation of the SDK.

That is, the SDK generation looks for the configuration file, finds the tag to be used, finds the swagger files and uses them to generate the SDK. That version of SDK will have the code from those specifically picked swagger files. 

What will happen if you have a swagger file that is not referenced in the configuration file? That swagger file will never get picked up and thus never be used in the SDK generation. 

Now, Where do you need to add the file? It depends on your use case. Do you have other swagger files also that need to be used in the generation of the SDK? If so, all of them must go inside one tag. Else, create a tag with this file alone and mark that as default tag. It really depends on your requirement. If you still have any questions, I am always here to help and you could arrange a call.

Reply:
@sarangan12 Thanks a lot for detailed explanation! This makes sense. Now It seems that the only thing I need to add is to add the new *package-2017-preview* to the Tag element in the beginning (line 31). Could you please shed some light on the use of this element and can it accept comma separated values?

Reply:
@finiteattractor It accepts a single value. So it will be package-2017-05-preview

Reply:
Great! I updated the file to reflect this! Thanks!